### PR TITLE
{Misc.} Update CODEOWNERS - add backup owner for help

### PR DESCRIPTION
**Pull request by jiasli**
The current `CODEOWNERS` only has @msyyc for help-related files. In case of out-of-office, PRs requiring approval for help-related files (e.g. https://github.com/Azure/azure-cli/pull/16707) cannot be merged.

Note that 

```
src/azure-cli/azure/cli/command_modules/search/
```

doesn't have an owner, so 

```
src/azure-cli/azure/cli/command_modules/search/_help.py
```

will need approval from @msyyc.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -15,38 +15,38 @@
 /src/azure-cli/azure/cli/command_modules/acr/ @djyou @fengzhou-msft @yungezz
 /src/azure-cli/azure/cli/command_modules/acs/ @rjtsdl @fengzhou-msft
 /src/azure-cli/azure/cli/command_modules/advisor/ @Prasanna-Padmanabhan
+/src/azure-cli/azure/cli/command_modules/apim/ @kevinhillinger @jonlester
```
Comment (jiasli) [inline .github/CODEOWNERS:18]: Re-sorting lines alphabetically.

**Comment by yonzhan** [comment]
Misc

**Review by yonzhan — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -15,38 +15,39 @@
...
 /src/azure-cli/azure/cli/command_modules/network/ @jsntcy @kairu-ms @msyyc
-/src/azure-cli/azure/cli/command_modules/privatedns/ @jsntcy @kairu-ms @msyyc
 /src/azure-cli/azure/cli/command_modules/policyinsights/ @cheggert
+/src/azure-cli/azure/cli/command_modules/privatedns/ @jsntcy @kairu-ms @msyyc
 /src/azure-cli/azure/cli/command_modules/profile/ @jiasli @evelyn-ys @fengzhou-msft
 /src/azure-cli/azure/cli/command_modules/rdbms/ @arde0708 @Juliehzl @evelyn-ys
 /src/azure-cli/azure/cli/command_modules/resource/ @Juliehzl @zhoxing-ms
 /src/azure-cli/azure/cli/command_modules/role/ @jiasli @evelyn-ys @jsntcy @fengzhou-msft
-/src/azure-cli/azure/cli/command_modules/storage/ @Juliehzl @jsntcy @zhoxing-ms @evelyn-ys
+/src/azure-cli/azure/cli/command_modules/search/ @huangbolun @kairu-ms
```
Comment (jiasli) [inline .github/CODEOWNERS:46]: Add new code owner for `search` module.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**